pyproject.toml

```diff
@@ -35,17 +34,18 @@ classifiers = [
 poetry_installer = "tox_poetry_installer"
 
 [tool.poetry.extras]
-poetry = ["poetry"]
+poetry = ["poetry", "cleo"]
 
 [tool.poetry.dependencies]
-python = "^3.6.1"
-poetry = {version = ">=1.0.0,<1.2", optional = true}
-poetry-core = "^1.0.0"
+python = "^3.7"
+cleo = {version = "^1.0.0a5", optional = true, allow-prereleases = true}
+poetry = {version = "^1.2.0", optional = true}
+poetry-core = "^1.1.0"
 tox = "^3.8.0"
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
 bandit = "^1.6.2"
-black = { version = "^21.12b0", allow-prereleases = true, python = "^3.7" }
+black = "^22.3.0"
 blacken-docs = { version = "^1.8.0", python = "^3.7" }
 ipython = { version = "^7.18.1", python = "^3.7" }
```

Comment:
Now that the project relies on `^3.7` we can remove the `^3.7` markers from the dev-deps